lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -458,6 +458,7 @@ protected function transformReportbackItem($data) {
   protected function transformUser($data) {
     return array(
       'id' => $data->id,
+      'drupal_id' => $data->drupal_id,
```

Comment:
So I don't think we actually want to return the Drupal ID back in the API response. Drupal ID shouldn't mean anything to anyone using the api... is there a specific reason why we need this returned?

If we **do** need it, we could call it something agnostic to the framework, like `legacy_id` or something. But ideally we shouldn't be outputting this back in the JSON API Response...